### PR TITLE
Revert "feat: add tags to creator registration"

### PR DESCRIPTION
**Pull request by NdibeRaymond**
Reverts unstructuredstudio/zubhub#1082

there is an error in this PR that makes it impossible to register a new user. @yokwejuste you should look at this again and try to verify that it works before creating a PR again. I assumed you already did that so didn't test properly before merging. Possibly try to get the tests in `zubhub_backend/zubhub/creators/tests/test_views.py` to pass.

### CODE REVIEW

**Comment by yokwejuste** [comment]
Please this should work with @coderatomy new UI the signup

**Comment by NdibeRaymond** [comment]
> Please this should work with @coderatomy new UI the signup

It does not work. I just tested it now

File: zubhub_backend/zubhub/creators/views.py [hunk 1]
```diff
@@ -186,7 +188,6 @@ class RegisterCreatorAPIView(RegisterView):
             "location": "string",\n
             "bio": "",\n
             "subscribe": false\n
-            "creator_tags": "string",\n
```
Comment (NdibeRaymond) [inline zubhub_backend/zubhub/creators/views.py:189]: here you are specifying that `creator_tags` is a string, but you were treating it like an array everywhere else

File: zubhub_backend/zubhub/creators/serializers.py [hunk 2]
```diff
@@ -248,22 +247,13 @@ def validate_location(self, location):
...
-        return creator_tags
-
     def get_cleaned_data(self):
         data_dict = super().get_cleaned_data()
         data_dict['phone'] = self.validated_data.get('phone', '')
         data_dict['dateOfBirth'] = self.validated_data.get('dateOfBirth', '')
         data_dict['location'] = self.validated_data.get('location', '')
         data_dict['bio'] = self.validated_data.get('bio', '')
         data_dict['subscribe'] = self.validated_data.get('subscribe', '')
-        data_dict['creator_tags'] = self.validated_data.get('creator_tags', [])
```
Comment (NdibeRaymond) [inline zubhub_backend/zubhub/creators/serializers.py:266]: here you treat `creator_tags` as an array

File: zubhub_backend/zubhub/creators/serializers.py [hunk 3]
```diff
@@ -198,8 +199,6 @@ class CustomRegisterSerializer(RegisterSerializer):
                                             queryset=Location.objects.all())
     bio = serializers.CharField(allow_blank=True, default="", max_length=255)
     subscribe = serializers.BooleanField(default=False)
-    creator_tags = serializers.SlugRelatedField(slug_field='name',
```
Comment (NdibeRaymond) [inline zubhub_backend/zubhub/creators/serializers.py:201]: the syntax here is basically saying `Creator.objects.all().filter(name=<value of creator_tags coming from the frontend>)[0], or something like that`. But the value of the creator_tags coming from the frontend given the way we are trying to extract it and the default we are providing, is an array, so won't work

**Comment by NdibeRaymond** [comment]
from the reviews above, you can see that you are mixing the `creator_tags` field. Treating it as an array in one place and treating it as string in others

**Pull request merged**